### ISSUE TITLE
Impossible to close the privacy modal with keyboard

### ISSUE THREAD

**Issue by radinamatic** [learningequality/kolibri#4973]
### Observed behavior

When navigating by keyboard user is prevented to close the *Usage and privacy* modal: even when the <kbd>Close</kbd> button is in focus, and the user presses the <kbd>Enter</kbd> key, the modal keeps reopening, until the user closes by mouse click (if they can).

#### Chrome
![modal-close-keyboard-chrome](https://user-images.githubusercontent.com/1457929/52496754-24aab200-2bcc-11e9-8226-7eab54b1168f.gif)

#### Firefox
![modal-close-keyboard-ff](https://user-images.githubusercontent.com/1457929/52496760-2a07fc80-2bcc-11e9-8cd6-7c96313a1672.gif)

*Note the additional focusable element in Firefox (the whole field)* - **should get the `focusable=false` attribute.**

### Expected behavior

Modal should be fully under control by keyboard only.

### User-facing consequences

Being blocked and unable to close the modal.

### Errors and logs
--

### Steps to reproduce

Navigate the *Usage and privacy* modal by keyboard and try to close

### Context

 * Kolibri version: 0.12.0.b3
 * Operating system: Windows 7
 * Browser: Chrome & Firefox

**Comment by indirectlylit** [comment]
(removing performance tag - I think that's typically reserved for 'slowness')

**Comment by radinamatic** [comment]
> performance tag typically reserved for 'slowness'

Noted! 👍

**Comment by jonboiser** [comment]
#4981
